core/CMake_symbol_check.cmake

```diff
@@ -47,8 +66,22 @@ execute_process(COMMAND
   ERROR_VARIABLE readelf_error
   OUTPUT_VARIABLE output
   )
+
+is_i3450_binutils_version(is_i3450)
+
+# XXX i#3450: Check for binutils/readelf version 2.30, and if yes, then ignore errors
+# for 32-bit release build. Binutils bug #24382 has been filed. Both earlier and later
+# versions seem ok.
+if (X86 AND NOT X64)
+  if (NOT DEBUG)
+    if (is_i3450)
+      set(readelf_error "")
+    endif ()
+  endif ()
+endif ()
 if (readelf_result OR readelf_error)
   message(FATAL_ERROR "*** ${READELF_EXECUTABLE} failed: ***\n${readelf_error}")
+endif ()
```

Comment:
This looks like a bug: a syntax error: an extra endif.  Plus it displaces the endif with the condition in the parens which will also be a syntax error.

Reply:
Thanks. Interesting that cmake tolerated it!